### DISCUSSION TITLE
How to use Preview option?

### DISCUSSION THREAD

**Discussion by pedjas** [gpx-animator/gpx-animator#597] [Q&A]
I have set project and when I click Preview button it shows computing something and that is it. It does nto show preview not shows file name if preview is saved in a file.

I checked folder where output is set to go to, no preview file there.

When I start rendering it renders output file properly, but it takes lot of time. I prefer having quickly generated preview so I can check if everything is properly set before starting render.

How is Preview supposed to be used?

**Comment by pentool** [comment]
In Preferences, make sure "Enable Preview" is selected.

**Comment by McPringle** [comment]
Sorry for my late reply, the last few months I took a sabbatical and rode my bicycle through the whole US. I hope you did not stop using GPX Animator in the meantime!

It sounds like a bug. As @pentool mentioned, please open the preferences, enable the preview setting, and check if the preview button works again. The preview button should work anyway, even without having the preview setting activated.

**Reply by McPringle** [reply]
I used GPX Animator v1.8.2 and the preview button works as intended with and without activated preview setting. Please check if you use the latest version (menu _Help > About_) and update, if not. If your problem persists, please provide more details for me to reproduce your problem:

- Which version of GPX Animator are you using?
- How did you install GPX Animator?
- Which Operating System and version are you using?
- Please provide the protocol (menu _Help > Protocol_)
- Please provide your settings (menu _File > Save as_)
- Please provide your GPX file(s)

If you want to keep your files private and not upload them here on GitHub, you can send them using email (GPG is supported) to my private address: marcus@fihlon.swiss

**Reply by pedjas** [reply]
Hm.. now I tried and it works as expected.

**Comment by pentool** [comment]
> Sorry for my late reply, the last few months I took a sabbatical and rode my bicycle through the whole US. 

Oooooo that sounds fantastic! Any videos of the journey?
